### PR TITLE
Cookies alignment breadcrumb fix

### PR DESCRIPTION
**Pull request by cedrickisemadigitalhmrc**
Addresses two issues

- The cookies banner - please can it be left-aligned with the content below it. 
- The words and the left-most button should be aligned with the gov.uk crown
- The cookies page has two breadcrumbs and a few alignment issues

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -14,19 +14,6 @@ def a_z_index(letter = 'a')
     }.join.html_safe
   end
 
-  def breadcrumbs
```
Comment (willfish) [inline app/helpers/application_helper.rb:17]: I guess this is not used?

**Comment by willfish** [comment]
You okay to paste some before/after screenshots? Code looks fine to me :)

File: app/webpacker/src/stylesheets/_cookie_banner.scss [hunk 2]
```diff
@@ -15,6 +15,7 @@
 .app-cookie-banner__message {
   margin: 0;
   @include govuk-width-container;
+  max-width: 1145px;
```
Comment (tavyy) [inline app/webpacker/src/stylesheets/_cookie_banner.scss:18]: Wondering why this size exactly?

File: app/views/pages/cookies.html.erb [hunk 3]
```diff
@@ -1,172 +1,183 @@
...
+<% content_for :before_main_content do %>
+  <%= generate_breadcrumbs(
+          "Cookies on UK Global Online Tariff",
+          [
+              [t('breadcrumb.home'), root_path],
+          ]
+      )
+  %>
+<% end %>
+<div class="cookies-settings">
```
Comment (willfish) [inline app/views/pages/cookies.html.erb]: I can't see this defined anywhere

**Review by tavyy — APPROVED** [review]

**Review by willfish — APPROVED** [review]
Checked this out locally and can see that it works

**Pull request merged**